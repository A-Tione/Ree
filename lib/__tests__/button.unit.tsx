import React from 'react'
import {render} from "@testing-library/react";
import '@testing-library/jest-dom'
import Button from '../button'

jest.mock('../icon/importIcons', () => {
    return {
      context: () => {
        const keys = () => [];
        const module = (key: string) => null;
        return {
          keys,
          module,
        };
      }
    };
  });

describe('button', () => {
    it('是个 div', () => {
        const json = render(<Button />).container
        expect(json).toMatchSnapshot()
    })
})