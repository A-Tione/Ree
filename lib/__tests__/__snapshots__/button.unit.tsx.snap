// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button 是个 div 1`] = `
<div>
  <svg
    className="ree-icon"
  >
    <use
      xlinkHref="#wechat"
    />
  </svg>
  <svg
    className="ree-icon"
  >
    <use
      xlinkHref="#apple"
    />
  </svg>
  <svg
    className="ree-icon testClassName"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <use
      xlinkHref="#apple"
    />
  </svg>
</div>
`;
